frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -248,6 +269,27 @@ const VirtualMachinesPage: React.FC<VirtualMachinesPageProps> = (props) => {
     },
   ];
 
+  const [importAllowed, setImportAllowed] = useSafetyFirst(false);
+
+  React.useEffect(() => {
+    VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAMESPACES.forEach((configMapNamespace) => {
+      checkAccess({
+        group: ConfigMapModel.apiGroup,
+        verb: 'get',
+        resource: ConfigMapModel.plural,
+        name: VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAME,
+        namespace: configMapNamespace,
+      })
+        .then((result: SelfSubjectAccessReviewKind) => {
+          if (result?.status?.allowed) {
+            setImportAllowed(result?.status?.allowed);
```

Comment:
can you replace `result?.status?.allowed` with `true` ?
we don't really using the `result` here, for example we can't set it to `false` in case the result is false

Reply:
it would be keeping the origin of that value. Nevertheless it doesn't really matter - changed to true